alt/asyncapi/README.md

```diff
@@ -0,0 +1,155 @@
+asyncapi
+========
+
+The AsyncAPI CLI
+
+[![oclif](https://img.shields.io/badge/cli-oclif-brightgreen.svg)](https://oclif.io)
+[![Version](https://img.shields.io/npm/v/asyncapi.svg)](https://npmjs.org/package/asyncapi)
+[![Downloads/week](https://img.shields.io/npm/dw/asyncapi.svg)](https://npmjs.org/package/asyncapi)
+[![License](https://img.shields.io/npm/l/asyncapi.svg)](https://github.com/asyncapi/cli/blob/master/package.json)
+
+<!-- toc -->
+* [Usage](#usage)
+* [Commands](#commands)
+<!-- tocstop -->
+# Usage
+<!-- usage -->
+```sh-session
+$ npm install -g @asyncapi/cli
+$ asyncapi COMMAND
+running command...
+$ asyncapi (-v|--version|version)
+@asyncapi/cli/0.1.0 darwin-x64 node-v14.15.1
+$ asyncapi --help [COMMAND]
+USAGE
+  $ asyncapi COMMAND
+...
+```
+<!-- usagestop -->
+# Commands
+<!-- commands -->
+* [`asyncapi context`](#asyncapi-context)
+* [`asyncapi context:add [CONTEXT-NAME] [ASYNCAPI-FILE-PATH]`](#asyncapi-contextadd-context-name-asyncapi-file-path)
```

Comment:
Commands are showing here with a colon (`:`). That's something we'll have to fix in this README. The rest of the code is working with spaces instead of colons.